apm-sniffer/optional-plugins/optional-spring-plugins/spring-annotation-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/annotations/SpringAnnotationConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.plugin.spring.annotations;
+
+import org.apache.skywalking.apm.agent.core.boot.PluginConfig;
+
+public class SpringAnnotationConfig {
+
+    public static class Plugin {
+        @PluginConfig(root = SpringAnnotationConfig.class)
+        public static class SpringAnnotation {
+            /**
+             * regex expression to match spring bean
+             */
+            public static String REGEX_EXPRESSION = ".*Service.*,.*Dao.*,.*Repository.*";
```

Comment:
```suggestion
            public static String REGEX_EXPRESSION = ".*Service$,.*Dao$,.*DAO$,.*Repository$";
```

Reply:
This should not You set this works in default, which could make users confused, because, once their classes are not matching these default rules, they will not be traced.

So, this should be default OFF.